go/libkb/merkle_client.go

```diff
@@ -943,10 +962,22 @@ func (mc *MerkleClient) storeRoot(m MetaContext, root *MerkleRoot) {
 	}
 }
 
+func (mc *MerkleClient) FirstSeqnoWithSkipsProd(m MetaContext) *keybase1.Seqno {
+	chk := GetMerkleCheckpoint(m)
+	var ret *keybase1.Seqno
+	if chk != nil {
+		ret = chk.Seqno()
+	}
+	if ret == nil || FirstProdMerkleSeqnoWithSkips > *ret {
+		ret = &FirstProdMerkleSeqnoWithSkips
+	}
+	return ret
+}
+
 func (mc *MerkleClient) FirstSeqnoWithSkips(m MetaContext) *keybase1.Seqno {
```

Comment:
`FirstSeqnoWithSkips` doesn't return the first seqno with skips, it returns the checkpoint. Name `FirstExaminableHistoricalRoot`?